### ISSUE TITLE
Official documentation not up to date with source code

### ISSUE THREAD

**Issue by phdesal** [sullo/nikto#576]
The official documentation does not seem in sync with the documentation in the source code.

As an example:
the `-404string` flag is documented in the source code: https://github.com/sullo/nikto/blob/master/documentation/sections/options.xml

the `-404string` flag is nowhere to be found in the official documentation: https://cirt.net/nikto2-docs/options.html

Anyway, thanks for a great tool!

**Comment by tmjvonboss** [comment]
i think the documentation is in sync with the last release, sadly outdated by 3 years XD

**Comment by tmjvonboss** [comment]
How does one help with documentation on Github Wiki? Ive never done that before

**Comment by sullo** [comment]
If you go to the Wiki link above (https://github.com/sullo/nikto/wiki) there's an Edit button the right. 

I stubbed out the section links copying from the 'old' documentation--red ones have no pages/content yet. I _believe_ the ones there are complete vs the old documentation with minor updates as I went. 

The main things needed are:
* copying 'old' text and re-formatting for the wiki
* updating things that are slightly dated
* adding missing things (as you noted)

Any help with any of those is _greatly_ appreciated.

For ref, the old doc is in the repo () in docbook format, as well as here:
https://cirt.net/nikto2-docs/

Thanks for asking @tmjvonboss !
